### PR TITLE
bitserializer: fix package info + simplify source renaming

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **bitserializer/all**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/bitserializer/0.10/conanfile.py [hunk 1]
```diff
@@ -51,7 +52,7 @@ def requirements(self):
         if self.options.with_cpprestsdk:
             self.requires("cpprestsdk/2.10.16")
         if self.options.with_rapidjson:
-            self.requires("rapidjson/1.1.0")
+            self.requires("rapidjson/cci.20200410")
```
Comment (prince-chrismc) [inline recipes/bitserializer/0.10/conanfile.py:55]: I think it's better if the default was an official release :thinking:

Comment (SSE4) [reply recipes/bitserializer/0.10/conanfile.py:55]: agree

Comment (SpaceIm) [inline recipes/bitserializer/0.10/conanfile.py:55]:
```suggestion
            self.requires("rapidjson/1.1.0")
```

Comment (SpaceIm) [reply recipes/bitserializer/0.10/conanfile.py]: ok, bump cancelled

**Review by SSE4 — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Pull request merged**